examples/config-dev.yaml

```diff
@@ -61,6 +61,7 @@ connectors:
 #     clientID: $GOOGLE_CLIENT_ID
 #     clientSecret: $GOOGLE_CLIENT_SECRET
 #     redirectURI: http://127.0.0.1:5556/dex/callback
+#     hostedDomain: $GOOGLE_HOSTED_DOMAIN
```

Comment:
I'm afraid this doesn't match the code. If I understand it correctly, it should be:
```
hostedDomains:
  - $GOOGLE_HOSTED_DOMAIN
```